barretenberg/cpp/src/barretenberg/stdlib/goblin_verifier/merge_recursive_verifier.cpp

```diff
@@ -24,11 +24,10 @@ MergeRecursiveVerifier_<CircuitBuilder>::MergeRecursiveVerifier_(CircuitBuilder*
  */
 template <typename CircuitBuilder>
 std::array<typename bn254<CircuitBuilder>::Element, 2> MergeRecursiveVerifier_<CircuitBuilder>::verify_proof(
-    const HonkProof& proof)
+    const StdlibProof<CircuitBuilder>& proof)
```

Comment:
Just bringing this in line with the other recursive verifiers which all receive a stdlib proof object